packages/mint-components/src/components/sqm-referral-table/sqm-referral-table.tsx

```diff
@@ -41,11 +48,23 @@ export class ReferralTable {
   /** @uiName Show Referred by user in table  */
   @Prop() showReferrer?: boolean = false;
 
+  /*
+   * Provide the column numbers (0 indexed) that should not be displayed in mobile views. Ex. 0,2,3
+   *
+   *  @uiName Mobile Hidden Columns  */
+  @Prop() hiddenColumns?: string = "0";
+
+  /** @uiName Mobile Breakpoint  */
+  @Prop() smBreakpoint?: number = 634;
```

Comment:
Why does the referral table have a different default mobile breakpoint than the rewards table?

Reply:
They are now all consistent with `sm-breakpoint="599px"` and `md-breakpoint="799px"`